### PR TITLE
[WIP] Feature/oas consistency

### PR DESCRIPTION
**Pull request by joeribekker**
Depends on: https://github.com/VNG-Realisatie/vng-api-common/pull/75

### CODE REVIEW

**Comment by sergei-maertens** [comment]
Backend tests seem to be failing on more than just the vng-api-common version that wasn't present yet

File: docs/index.rst [hunk 1]
```diff
@@ -1,19 +1,60 @@
-.. include:: ../README.rst
+=================
+zaaktypecatalogus
+=================
+
+:Version: 0.1.0
```
Comment (sergei-maertens) [inline docs/index.rst]: Version should be picked up via `.bumpversion.cfg`, but low prio for now

File: src/ztc/conf/base.py [hunk 2]
```diff
@@ -11,7 +14,7 @@
 BASE_DIR = os.path.abspath(os.path.join(DJANGO_PROJECT_DIR, os.path.pardir, os.path.pardir))
 
 # Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
```
Comment (sergei-maertens) [inline src/ztc/conf/base.py]:
```suggestion
# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
```

File: src/ztc/conf/base.py [hunk 3]
```diff
@@ -124,10 +122,10 @@
 WSGI_APPLICATION = 'ztc.wsgi.application'
 
 # Database: Defined in target specific settings files.
-# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+# https://docs.djangoproject.com/en/2.0/ref/settings/#databases
```
Comment (sergei-maertens) [inline src/ztc/conf/base.py]:
```suggestion
# https://docs.djangoproject.com/en/2.2/ref/settings/#databases
```

File: src/ztc/conf/base.py [hunk 4]
```diff
@@ -124,10 +122,10 @@
...
 
 # Password validation
-# https://docs.djangoproject.com/en/1.11/ref/settings/#auth-password-validators
+# https://docs.djangoproject.com/en/2.0/ref/settings/#auth-password-validators
```
Comment (sergei-maertens) [inline src/ztc/conf/base.py]:
```suggestion
# https://docs.djangoproject.com/en/2.2/ref/settings/#auth-password-validators
```

File: src/ztc/conf/base.py [hunk 5]
```diff
@@ -146,11 +144,11 @@
 
 
 # Internationalization
-# https://docs.djangoproject.com/en/1.11/topics/i18n/
+# https://docs.djangoproject.com/en/2.0/topics/i18n/
```
Comment (sergei-maertens) [inline src/ztc/conf/base.py]:
```suggestion
# https://docs.djangoproject.com/en/2.2/topics/i18n/
```

File: src/ztc/conf/base.py [hunk 6]
```diff
@@ -166,7 +164,7 @@
 )
 
 # Static files (CSS, JavaScript, Images)
-# https://docs.djangoproject.com/en/1.11/howto/static-files/
+# https://docs.djangoproject.com/en/2.0/howto/static-files/
```
Comment (sergei-maertens) [inline src/ztc/conf/base.py]:
```suggestion
# https://docs.djangoproject.com/en/2.2/howto/static-files/
```

**Comment by joeribekker** [comment]
Ondanks [WIP] label? Maar prima :)

**Pull request merged**